README.md

```diff
@@ -19,17 +23,28 @@ The analyses in the Machine Learning module are structured in JASP in the follow
        - Neural Network
        - Random Forest
        - Regularized Linear
+       - Support Vector Machine
     -- Classification
        - Boosting
        - Decision Tree
        - K-Nearest Neighbors
        - Linear Discriminant
        - Neural Network
        - Random Forest
+       - Support Vector Machine
     -- Clustering
        - Density-Based
        - Fuzzy C-Means
        - Hierarchical
        - Neighborhood-Based
        - Random Forest
+    -- Prediction
 ```
+
+### Translations
+
+The JASP translation project is located at the following [link](https://hosted.weblate.org/projects/jasp/). The Machine Learning module is referred to as `jaspMachineLearning` and consists of two distinct components: `jaspMachineLearning-QML` and `jaspMachineLearning-R`, both of which require translation. The participation of any individual who is interested would be highly valued. As demonstrated below, we have made substantial progress in the translation of the Audit module.
+
+| Interface | Results |
+| :---: | :---: |
+| [![image](https://hosted.weblate.org/widgets/jasp/-/jaspmachinelearning-qml/multi-auto.svg)](https://hosted.weblate.org/engage/jasp/) | [![image](https://hosted.weblate.org/widgets/jasp/-/jaspmachinelearning-r/multi-auto.svg)](https://hosted.weblate.org/engage/jasp/) |
```

Comment:
> As demonstrated below, we have made substantial progress in the translation of the Audit module.

shouldn't that be 

of the *Machine learning* module.

?

Reply:
True